src/main/java/cpw/mods/ironchest/ChestChangerType.java

```diff
@@ -81,32 +90,31 @@ public void addRecipes() {
 
     public static void buildItems(Configuration cfg) {
```

Comment:
One problem(that doesn't really effect GTNH) is that the Crystal/Obsidian -> Dark Steel upgrade items get added regardless of if dark steel chests are enabled or not. They need a case here similar to the DIAMONDDARKSTEEL one, to prevent the upgrade items from being added when dark steel chests are disabled.